johnsonandjohnson/Bodiless-JS
# fix(fclasses) Design transformer does not update passthroughProps when its props are changed

## Changes
* fixed an issue with updating design transformer props

## Test Instructions
* open a product page on test site
* click bv placeholder
* enter "BV Product External ID" (update product external id if it is already added)
* click save

actual: bv placeholder content is updated
expected: bv placeholder content is not updated
